README.md

```diff
@@ -5,6 +5,10 @@ This project currently lives in a pre-alpha status.  Our current release is not
 
 For more information, please, consult the documentation of the main [OpenTelemetry php project](https://github.com/open-telemetry/opentelemetry-php).
 
+## Issues
+
+Issues have been disabled for this repo in order to help main consistency between this repo and the main opentelemetry-php repo.  If you have an issue you'd like to raise about this issue, please use the [OpenTelemetry PHP Issue section](https://github.com/open-telemetry/opentelemetry-php/issues/new/choose).  Please prefix the title of the issue with [opentelemetry-php-contrib].
```

Comment:
I still stumble upon "issue [...] about this issue".
I have no idea, if that's correct English, but I find this wording a bit confusing.